hooks/pre-commit

```diff
@@ -9,3 +9,5 @@ SCRIPTS_DIR=$UNIVERSE_DIR/scripts
 
 $SCRIPTS_DIR/build.sh
 
+# Fail if there are unstaged changes
+git diff --exit-code
```

Comment:
This is pretty aggressive but I guess it is okay given how the build command works.